ui/src/components/Trade.tsx

```diff
@@ -4,13 +4,16 @@ import scrollIcon from '../assets/scroll.png';
 import istIcon from '../assets/IST.svg';
 import mapIcon from '../assets/map.png';
 import potionIcon from '../assets/potionBlue.png';
+import netflixLogo from '../assets/netflix.svg';
+import disneyLogo from '../assets/disney.svg';
+import hboLogo from '../assets/hbomax.svg';
+import primeLogo from '../assets/amazon.svg';
```

Comment:
I'm pretty sure these are registered trademarks, not to be used without permission. OK for a hackathon, but we can't keep the code around long.